package.json

```diff
@@ -69,14 +71,15 @@
     "eslint": "~8.15.0",
     "eslint-config-prettier": "8.1.0",
     "eslint-plugin-cypress": "2.12.1",
+    "fundamental-styles": "^0.26.5",
```

Comment:
We need this one to get the same array of themes supported by both libs - ui5 and fundamental-styles.